### ISSUE TITLE
Whats new textarea: after selecting media to upload the container hides

### ISSUE THREAD

**Issue by kjnanda** [rtMediaWP/rtMedia#1353]
```
Test Environment
WordPress: 5.2.1
BuddyPress: 4.1.0 ( legacy template )
rtMedia core: 4.5.5 
Theme: Default theme ( mainly occurs in 2012 theme )
```
**Steps to Reproduce**
1. Go to activity page
2. Select media to upload, observe that on selecting media, media hides. 

Refer: https://drive.google.com/open?id=1XivLIQ0ZscZFWHVeO5EW8aWDCAXDmDLZ
**Note: this issue occurs randomly.**

**Comment by surajkrsingh** [comment]
@kjnanda 

I have test this issue with given environment and also with the latest version of rtmedia v4.5.7.
I am not able to replicate this issue at my end and it is working  as expected.

**Comment by surajkrsingh** [comment]
PR:-https://github.com/rtMediaWP/rtMedia/pull/1388

**Comment by kjnanda** [comment]
Added changes are working as expected.

**Comment by kjnanda** [comment]
This is working as expected and tested with `pre-release` branch. ✅

**Comment by kjnanda** [comment]
This is working as expected and tested with `dev` branch. ✅

**Comment by kjnanda** [comment]
This is working as expected and tested with `master` branch. ✅

**Comment by kjnanda** [comment]
Closing this issue as this has been fixed and release in - V 4.5.8
